utils/Rector/RemoveErrorSuppressInTryCatchStmtsRector.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Utils\Rector;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\ErrorSuppress;
+use PhpParser\Node\Stmt\TryCatch;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+final class RemoveErrorSuppressInTryCatchStmtsRector extends AbstractRector
+{
+	public function getRuleDefinition(): RuleDefinition
+	{
+		return new RuleDefinition('Remove error suppress @', [
+			new CodeSample(
+				<<<'CODE_SAMPLE'
+try {
+	@rmdir($dirname);
+} catch (Exception $e) {}
+CODE_SAMPLE
+,
+				<<<'CODE_SAMPLE'
+try {
+	rmdir($dirname);
+} catch (Exception $e) {}
+CODE_SAMPLE
```

Comment:
since we're on PHP 7.3 minimum, we can use indented heredocs.

Reply:
updated.